required_scripts/summarize_Ig_50d8a94.R

```diff
@@ -21,25 +24,60 @@ option_list <- list(
               help="Flat file containing Ig Calls using Gammit"),
   make_option(c("-c", "--count"), action="store",type="integer", default=2,
               help="Minimum caller count [default %default]",
-              metavar="number")
+              metavar="number"),
+  make_option(c("--genes"), action="store",
+              help="Minimum caller count [default %default]")
```

Comment:
No default value is defined but it is used in the help.  Do we need to add default=c("G1","G2","G3","G4")

Reply:
Fixed this message now, no default needed here (else we require the user to pass in a comma separated list). But this option is defined to use an input file.